src/client/routes/Vaults/index.tsx

```diff
@@ -207,21 +211,13 @@ export const Vaults = () => {
   return (
     <ViewContainer>
       <StyledSliderCard
-        header={t('vaults:your-time-card.header')}
+        header="Goblin town or the Citadel?"
         Component={
           <Text>
-            <p>{t('vaults:your-time-card.desc-1')}</p>
-            <p>{t('vaults:your-time-card.desc-2')}</p>
-            <p>{t('vaults:your-time-card.desc-3')}</p>
+            <p>Wherever we’re going, relax and earn yield</p>
           </Text>
         }
-        // NOTE Example for slideshow array
-        // slidesContent={[
-        // {
-        //   header: 'Test header',
-        //   content: 'test content',
-        // },
-        // ]}
+        background={<img src={GoblinTown} alt="Goblin town or the Citadel?"></img>}
```

Comment:
nit: We don't need to close an `img` tag
```suggestion
        background={<img src={GoblinTown} alt="Goblin town or the Citadel?">}
```